packages/config/config/devices/0x010b/dn3g6ja082.json

```diff
@@ -0,0 +1,49 @@
+// Sharp DN3G6JA082
+// Open/Close Sensor
+{
+	"manufacturer": "Sharp",
+	"manufacturerId": "0x010b",
+	"label": "DN3G6JA082",
+	"description": "Open/Close Sensor",
+	"devices": [
+		{
+			"productType": "0x4761",
+			"productId": "0x0082",
+			"zwaveAllianceId": 2979
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"2": {
+			"label": "Enable Switch",
+			"description": "Enable switch of \"alarm notification\" report",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Open/close off",
```

Comment:
Not sure what this means. Which is on, opened or closed? Please use the format "On when opened" and "Off when opened"

Reply:
Looks like it's for toggling an "Alarm Notification" report on open/close.